draft-ietf-quic-v2.md

```diff
@@ -140,8 +140,17 @@ Compatible version negotiation between versions 1 and 2 follow the same
 requirements in either direction. This section uses the terms "original
 version" and "negotiated version" from {{QUIC-VN}}.
 
-If the server elects to send a Retry packet, it MUST do so using the original
-version.
+If the server sends a Retry packet, it MUST use the original version. The client
```

Comment:
I think that you want to say what the client does with a Retry in a version other than the original: it ignores it.